### DISCUSSION TITLE
Testing liquidity

### DISCUSSION THREAD

**Discussion by tipogi** [ACINQ/phoenixd#37] [Q&A] [answered]
Trying to understand how does it work the `--auto-liquidity` option, I have came across the __max fee credit__ error and I do not know how to go forward. All the payments get rejected and not any new channel opens. This is the actual state of the channels

```bash
./phoenix-cli getinfo
```
```json
{
    "nodeId": "021bea6e91e823b237c10529837522e7c0eafb999ea8a491cdaadc76d7b1177e20",
    "channels": [
        {
            "state": "Closed",
            "channelId": "5e691e41c366e34e6872b7894b3efce8d2a8962ecfef0a9029eae72aaceaea5a",
            "balanceSat": 893,
            "inboundLiquiditySat": 15124903,
            "capacitySat": 15127385,
            "fundingTxId": "ee8f27bd0ae4c3f6c59496db8d20bee38061a43cfaa741981d15d8dfc40bf98b"
        },
        {
            "state": "Closing",
            "channelId": "11cc8d78c5f9f18492c13e3856575f16f0ae5e2eb04ee9600cbcaa46206386eb",
            "balanceSat": 618,
            "inboundLiquiditySat": 12658091,
            "capacitySat": 12660297,
            "fundingTxId": "a0600c0b0bf1a3a9b0f3328601b831431c3f1a6644d18168360c177de8b742af"
        }
    ],
    "chain": "testnet",
    "version": "0.1.4-04bd430"
}
```

```bash
./phoenix-cli getbalance
```

```json
{
    "balanceSat": 0,
    "feeCreditSat": 100000
}
```

The __phoenixd__ service has the following output
```bash
lightning payment rejected (amount=10000 sat): over max fee credit (max=100000 sat)
```

**Comment by pm47** [comment, marked as answer]
The [cost of liquidity is 1% + mining fees](https://phoenix.acinq.co/server/auto-liquidity#fees), with mining fees limited themselves to 1% of the liquidity amount by default. This brings the effective receive fee to 2% at the maximum by default. If mining fees are over the configured budget, then the payment will be added to the fee credit. If the fee credit reaches its max value, then payments start getting rejected. That's what happened here due to the current mining fee spike.

You can increase the budget for mining fees with `--max-mining-fee` if you don't mind paying more. Alternatively, if you choose a larger value for `--auto-liquidity`, then the budget for mining fees is automatically higher.

**Comment by tipogi** [comment]
Yes, you are right. Great job! thanks for your time, really appreciate
